corehq/apps/change_feed/tests/test_pillow.py

```diff
@@ -36,39 +36,16 @@ def setUp(self):
                 bootstrap_servers=settings.KAFKA_BROKERS,
                 consumer_timeout_ms=100,
             )
-        self.pillow = get_change_feed_pillow_for_db('fake-changefeed-pillow-id', self._fake_couch)
-
-    def test_process_change(self):
-        document = {
-            'doc_type': 'CommCareCase',
-            'type': 'mother',
-            'domain': 'kafka-test-domain',
-        }
-        self.pillow.process_change(Change(id='test-id', sequence_id='3', document=document))
-        message = next(self.consumer)
-
-        change_meta = change_meta_from_kafka_message(message.value)
-        self.assertEqual(SOURCE_COUCH, change_meta.data_source_type)
-        self.assertEqual(self._fake_couch.dbname, change_meta.data_source_name)
-        self.assertEqual('test-id', change_meta.document_id)
-        self.assertEqual(document['doc_type'], change_meta.document_type)
-        self.assertEqual(document['type'], change_meta.document_subtype)
-        self.assertEqual(document['domain'], change_meta.domain)
-        self.assertEqual(False, change_meta.is_deletion)
-
-        with self.assertRaises(ConsumerTimeout):
+        try:
+            # This initialized the consumer listening from the latest offset
             next(self.consumer)
+        except StopIteration:
+            pass
+        self.pillow = get_change_feed_pillow_for_db('fake-changefeed-pillow-id', self._fake_couch)
 
-    def test_process_change_with_unicode_domain(self):
-        document = {
-            'doc_type': 'CommCareCase',
-            'type': 'mother',
-            'domain': 'हिंदी',
-        }
-        self.pillow.process_change(Change(id='test-id', sequence_id='3', document=document))
-        message = next(self.consumer)
-        change_meta = change_meta_from_kafka_message(message.value)
-        self.assertEqual(document['domain'], change_meta.domain)
```

Comment:
it wasn't totally obvious to me why these two tests were removed. are they redundant?

Reply:
I don't know. it was probably accidental when I was working on fixing them